plugins/AuctionMaster/auctions_manager.yml

```
@@ -0,0 +1,305 @@
+#This is the lore of every auctioned item in the menu:
+#Use %amount% for item's amount
+auction-item-lore:
+  #The lore for items with no bids on them
+  #Use %display-name-seller% for display name of the seller
+  #Use %starting-bid% for the starting bid
+  #Use %duration% for auction duration
+  auction-item-no-bid:
+    - '&8&m---------------------'
+    - '&7Vendedor: %display-name-seller%'
+    - '&7La puja empieza en: &6%starting-bid% monedas'
+    - ''
+    - '&7Acaba en: &e%duration%'
+    - ''
+    - '&eClick para inspeccionar!'
+  #The lore for items with bids on them
+  #Use %display-name-seller% for display name of the seller
+  #Use %bids% for the amount of bids
+  #Use %top-bid% for the top bid
+  #Use %top-bid-player% for the top bid player
+  #Use %duration% for auction duration
+  auction-item-with-bids:
+    - '&8&m---------------------'
+    - '&7Vendedor: %display-name-seller%'
+    - '&7Pujas: &a%bids% pujas'
+    - ''
+    - '&7Mejor oferta: &6%top-bid% monedas'
+    - '&7Mejor ofertador: %top-bid-player%'
+    - ''
+    - '&7Acaba en: &e%duration%'
+    - ''
+    - '&eClick para inspeccionar!'
+  #Use %display-name-seller% for display name of the seller
+  #Use %bids% for the amount of bids
+  #Use %top-bid% for the top bid
+  #Use %top-bid-player% for the top bid player
+  auction-item-ended:
+    - '&8&m---------------------'
+    - '&7Vendedor: %display-name-seller%'
+    - '&7Pujas: &a%bids% bids'
+    - ''
+    - '&7Mejor oferta: &6%top-bid% monedas'
+    - '&7Mejor ofertador: %top-bid-player%'
+    - ''
+    - '&7Estado: &aEnded!'
+    - ''
+    - '&eClick para inspeccionar!'
+  #Use %display-name-seller% for display name of the seller
+  #Use %starting-bid% for the starting bid
+  auction-item-expired:
+    - '&8&m---------------------'
+    - '&7Vendedor: %display-name-seller%'
+    - '&7La puja empieza en: &6%starting-bid% monedas'
+    - ''
+    - '&7Estado: &cCaducado!'
+    - ''
+    - '&eClick para inspeccionar!'
+
+sort-auction-item: '154'
+sort-auction-item-name: '&aClasificar'
+#The lore is dynamic so you can only edit
+#the names of the sortings
+sort-auction-item-sorting:
+  highest-bid: 'Oferta mas alta'
+  lowest-bid: 'Oferta mas baja'
+  ending-soon: 'Acabando en poco'
+  most-bids: 'Mayoría de las ofertas'
+  click-to-switch: '&eHaz click para cambiar el orden!'
+
+search-auction-item: '323'
+search-auction-item-name: '&aBuscar'
+search-auction-item-lore:
+  - '&7Encuentra items por su nombre.'
+  - ''
+  - '&eClick para buscar!'
+
+#The message displayed on the sign when searching for an auction
+#There are only 3 rows because the first is where the user writes
+search-sign-message:
+  - '^^^^^^^^^^^^^^^'
+  - 'Escriba su'
+  - 'busqueda'
+
+create-menu-name: '&8creador de pujas'
+manage-own-auctions-menu-name: '&8tu gestor de acciones'
```

¿Deberían de ser mayúsculas? Ah, y "auctions" es "subastas"
```suggestion
create-menu-name: '&8Creador de pujas'
manage-own-auctions-menu-name: '&8Tu gestor de subastas'
```